ckanext/dia/templates/uris/new.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'page.html' %}
+{% import 'macros/form.html' as form %}
+
+{% block subtitle %}{{ _('Mint a new URI') }}{% endblock %}
+
+{% block secondary_content %}
+  <section class="module module-narrow module-shallow">
+    <h2 class="module-heading"><i class="fa fa-info-circle"></i> {{ _('URIs for use in linked datasets') }}</h2>
+    <div class="module-content">
+        <p>
+            {% trans %}
+                Some info about what uri minting is and the DIA defined structure
```

Comment:
Is this to be filled in later?

Reply:
Yes, I was thinking to do that today, likely to get some feedback around this sort of thing once it's on UAT